Assets/Scripts/UI/PickupPup/Currency/CurrencyPanel.cs

```diff
@@ -119,10 +124,13 @@ void updateCoinsDisplay(int newAmount)
 
     void updateDogFoodDisplay(int newAmount)
     {
-        dogFoodDisplay.updateAmount(newAmount);
+        foreach(CurrencyDisplay display in dogFoodDisplays)
```

Comment:
I don't think this is correct. This logic looks like it's updated all of the food displays to the same value each time the value changes